src/trackers/classes/commonRequest.js

```diff
@@ -99,8 +103,10 @@ function _finalize (request, totalSites) {
     request.fpStd = Math.sqrt(sumSquare / (request.fpPerSite.length - 1)) || 0
 
     delete request.fpPerSite
+
+    request.exampleSites = getExampleSites([...request.pages], sharedData.config.includeExampleSites)
+    delete request.pages
     
-    request.pages = [...request.pages]
```

Comment:
This removes the `pages` array from a `commonRequest`. Was that intended?

Reply:
Yes, I use it for debugging sometimes, but otherwise it is not used.